### PR TITLE
[Improvement-12945][Tools] improvement: fix some sonarlint warning and file name spell error

### PR DESCRIPTION
**Pull request by ZhongJinHacker**
## Purpose of the pull request
fix issue #12945 

improvement  : fix some sonarlint warning and file name spell error

## Brief change log
please check the code diff

## Verify this pull request

*Manually verified the change by testing locally.*

### CODE REVIEW

**Comment by davidzollo** [comment]
thanks for your first contribution to Apache DolphinScheduler, It makes a lot of sense,Looking forward to your deep participation

this PR looks good to me. and please wait the CI check result

File: dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/demo/ProxyProcessDefinitionController.java [hunk 1]
```diff
@@ -71,7 +70,7 @@ public ProxyResult createProcessDefinition(String token,
         } catch (IOException e) {
             throw new RuntimeException(e);
         }
-        proxyResult = JSONUtils.parseObject(responseBody, ProxyResult.class);
+        ProxyResult proxyResult = JSONUtils.parseObject(responseBody, ProxyResult.class);
```
Comment (Radeity) [inline dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/demo/ProxyProcessDefinitionController.java:73]:
```suggestion
        return JSONUtils.parseObject(responseBody, ProxyResult.class);
```

File: dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/demo/ProxyProcessDefinitionController.java [hunk 2]
```diff
@@ -71,7 +70,7 @@ public ProxyResult createProcessDefinition(String token,
...
 
         return proxyResult;
```
Comment (Radeity) [inline dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/demo/ProxyProcessDefinitionController.java:75]:
```suggestion
```

**Review by Radeity — CHANGES_REQUESTED** [review]

File: dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/demo/ProxyProcessDefinitionController.java (hunk 2 above)
Comment (Radeity) [inline dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/demo/ProxyProcessDefinitionController.java]:
```suggestion
        return JSONUtils.parseObject(responseBody, ProxyResult.class);
```

**Review by Radeity — CHANGES_REQUESTED** [review]

**Review by EricGao888 — DISMISSED** [review]

**Comment by EricGao888** [comment]
@ZhongJinHacker Could u plz run 'mvn spotless:apply' to fix formatting errors so that CI could pass? Thanks.

**Comment by ZhongJinHacker** [comment]
@EricGao888  ok, I had run 'mvn spotless:apply' in my local PC today,  evertything is ok， and then I had pushed the fixed code. please review again. thank you for your suggestion 
dpsder001

**Comment by ZhongJinHacker** [comment]
> SonarCloud Quality Gate failed.    [Quality Gate failed](https://sonarcloud.io/dashboard?id=apache-dolphinscheduler&pullRequest=12946)
> 
> [Bug](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=BUG) [A](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=BUG) [0 Bugs](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=CODE_SMELL) [1 Code Smell](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=12946&resolved=false&types=CODE_SMELL)
> 
> [](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=12946&metric=new_coverage&view=list) [0.0% Coverage](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=12946&metric=new_coverage&view=list) [](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=12946&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=12946&metric=new_duplicated_lines_density&view=list)

fix

**Review by Radeity — DISMISSED** [review]

**Comment by Radeity** [comment]
@ZhongJinHacker Please add license header for two added UT file.

**Comment by Radeity** [comment]
@ZhongJinHacker Please add license header for two UT files.

**Comment by ZhongJinHacker** [comment]
@Radeity fix, thank you

**Comment by ZhongJinHacker** [comment]
I have add unit tests to fix unittest coverage problem.
tips:  some source code is hard to write unit test like DolphinSchedulerManager.java which need a real database datasource access in its constuctor method. It needs  overrload a new constuctor method for DolphinSchedulerManager. 
 @EricGao888

**Comment by ZhongJinHacker** [comment]
@EricGao888 @davidzollo could you merge this PR ? all checks have passed. I can't wait to do something for the next issue

**Pull request closed without merging**